src/functions/certGenInit.ts

```diff
@@ -18,15 +19,19 @@ const certGenInit: Handler = async (event: any, context?: Context, callback?: Ca
     }
 
     // Convert the received event into a readable array of filtered test results
-    const records: any[] = StreamService.getTestResultStream(event);
+    const expandedRecords: any[] = StreamService.getTestResultStream(event);
+    const certGenfilteredRecords: any[] = Utils.filterCertificateGenerationRecords(expandedRecords);
```

Comment:
certGenfilteredRecords should really be certGenFilteredRecords (capital F)

Reply:
Done